readme.md

```diff
@@ -70,6 +70,25 @@ pify(request, {multiArgs: true})('https://sindresorhus.com').then(result => {
 });
 ```
 
+##### allArgs
+
+Type: `boolean`
+Default: `false`
+
+By default, the promisified function will only return the second argument from the callback. Turning this on will make it return an array of all arguments from the callback, *including* the error argument. This can be useful if you want to avoid a try-catch block when using async-await.
```

Comment:
It's not up to me to decide, but it just doesn't feel right with promises. I'd patch `request` like this if I were you:

```js
try {
	const [httpResponse, body] = pify(request, {multiArgs: true})('https://sindresorhus.com');

	if (httpResponse.status !== 200) {
		throw new Error('Response status isn\'t 200');
	}

	console.log(body);
} catch (err) {
	console.log('Error!');
}
```

I wonder what @sindresorhus has to say.

Consider using [`got`](https://github.com/sindresorhus/got) though. It supports promises OOTB :wink: